models/Element/AbstractElement.php

```diff
@@ -732,4 +735,67 @@ protected function renewInheritedProperties(): void
         $inheritedProperties = $this->getDao()->getProperties(true);
         $this->setProperties(array_merge($inheritedProperties, $myProperties));
     }
+
+    protected function retryableFunction(?callable $beforeRetryables = null, ?callable $retryableFunc = null,
+        ?callable $onCommit = null, ?callable $onBeforeRetry = null, ?callable $onFailure = null,
+        int $maxRetries = 5): void
+    {
```

Comment:
The parameters are all typed as `callable`, but are only called if you pass a `Closure`.
Therefore, I suggest using this as the parameter type:

```suggestion
    protected function retryableFunction(
        ?Closure $beforeRetryables = null, 
        ?Closure $retryableFunc = null,
        ?Closure $onCommit = null, 
        ?Closure $onBeforeRetry = null, 
        ?Closure $onFailure = null,
        int $maxRetries = 5,
    ): void {
```